python/sdk/tests/test_build.py

```diff
@@ -111,8 +111,10 @@ def test_list_tests(self, unwrap):
 
     def test_update_test(self, unwrap):
         updated_name = 'updated_test'
-        res = unwrap(self.build.update_test)(self.build, test_id=pytest.test_id, name=updated_name, technique='T1234.001')
+        res = unwrap(self.build.update_test)(
+            self.build,test_id=pytest.test_id, name=updated_name, technique='T1234.001', crowdstrike_expected_outcome=EDRResponse.PREVENT)
```

Comment:
that doesn't look very alphabetical! Or do we not do that for arg lists? Also missing space between build,test_id

Reply:
shame on me. fixed